pkg/operator/resource/resourcemerge/generic_config_merger.go

```diff
@@ -133,6 +134,13 @@ func MergePrunedProcessConfig(schema runtime.Object, specialCases map[string]Mer
 
 type MergeFunc func(dst, src interface{}, currentPath string) (interface{}, error)
 
+var _ MergeFunc = RemoveConfig
+
+// RemoveConfig is a merge func that elimintes an entire path from the config
+func RemoveConfig(dst, src interface{}, currentPath string) (interface{}, error) {
```

Comment:
path is misleading. I thought you follow the path and cut it out.

Reply:
See type assertion above.  That is the signature of the method.  See usage here; https://github.com/openshift/cluster-kube-apiserver-operator/blob/master/pkg/operator/targetconfigcontroller/targetconfigcontroller.go#L223